jsonapi/jsonapi_utility_blackbox_test.go

```diff
@@ -78,3 +78,30 @@ func TestErrorToJSONAPIError(t *testing.T) {
 	require.Equal(t, jsonapi.ErrorCodeUnknownError, *jerr.Code)
 	require.Equal(t, strconv.Itoa(httpStatus), *jerr.Status)
 }
+
+func ExampleFormatMemberName() {
```

Comment:
Shouldn't this be a test?

Reply:
Examples are tests ;):

> Godoc examples are snippets of Go code that are displayed as package documentation and that are verified by running them as tests.

(source: https://blog.golang.org/examples)

See also: https://golang.org/pkg/testing/#hdr-Examples

Reply:
Thanks @kwk . I didn't know this ;)